### DISCUSSION TITLE
How `Window->Renderer` affect font rendering?

### DISCUSSION THREAD

**Discussion by ghost** [SpartanJ/ecode#162] [General]
I know changing the render's settings will affect the whole UI, not just font rendering. But I want to know if it could improve font rendering quality. This is out of curiosity, as after increasing the font size, the rendering of the code editor's font is much better for me.

**Comment by SpartanJ** [comment]
It does not affect. The renderer just use a different code path to render on screen but the final result should be exactly the same (not only for the fonts, but the whole UI). I recommend you to find a font that fits best for you, there are really cool small fonts for smaller screens, maybe you can use something like [proggy](https://github.com/bluescan/proggyfonts).
